### PR TITLE
add CSS padding to speakers list

### PR DESCRIPTION
**Pull request by mbvgua**
resolve issue #369, allowing for padding in speakers ul in the conference page

### CODE REVIEW

**Comment by kjaymiller** [comment]
Couple of thoughts on this on @mbuguadouglas, but first thank you for putting in the work for this. 

The original issue was to do this for all `li` objects on the site that are in `article` tags. This would create a consistency across the website. 

Your change only applies to the speaker list.

I did some digging. Here are some things that we will need to verify:

- What templates wrap content in the `<article></article>`. Should more or less do this. 
  **For You** Can you verify that the template that the `conferences page` along with the other pages like `about.html` and `open-source-program.html` also are wrapped in `<article>` tags?
  
  - Can you replace your class with a more broad `article  > li` tag that would apply this across the website?
  
#### Bonus points.:
 
 Changes should be tested. I would love a pytest that uses playwright to evaluate the CSS values. - Learn more at - https://playwright.dev/docs/evaluating
 
 If you aren't able to do this, or need help we can file an issue and I'll be happy to work with you on this (Perhaps you can ping me in the BPD discord)

**Review by kjaymiller — CHANGES_REQUESTED** [review]
See Comment

**Comment by mbvgua** [comment]
Thanks for the prompt feedback @kjaymiller . And I think i get what youre saying in the first part, so Ill work on that. For the second part(on testing), i do not know much about that :laughing: but id be happy to learn along the way. Using the resource assigned, ill be sure to contact you incase of any issues.

**Comment by kjaymiller** [comment]
I definitely get it. Let's get the change added and we can get the test implementation as another issue.

**Comment by mbvgua** [comment]
Handled the CSS stying bit thus resolving issue #369 .However the resulting issue brought up in #400 about playwright testing proved abit difficult to implement in code and I might need some more time to figure it out.

File: _includes/conferences.html [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+    {% endif %}
+    {% if conference.speaking != "_No response_" %}
+      <h3>Speaking:</h3>
+      <li>
       {{ conference.speaking | markdownify }}
-    </ul>
-  {% endif %}
+      </li>
+    {% endif %}
+  </article>
```
Comment (kjaymiller) [inline _includes/conferences.html]: try this 

```
<div>
{% for conference in site.data.conferences %}
  <article class="conference">
    <h2>
      <a href="{{ conference.url }}">{{ conference.name }}</a> ({{ conference.dates }}) {% if conference.location != "_No response_" %}- {{ conference.location }}{% endif %}
    </h2>
    <p>Type: {{ conference.type }}</p>
    {% if conference.summary != "_No response_" %}
      {{ conference.summary | markdownify }}
    {% endif %}
    {% if conference.speaking != "_No response_" %}
      <h3>Speaking:</h3>
          {{ conference.speaking | markdownify }}
    {% endif %}
  </article>
    {% endfor %}
{% else %}
<b>No conferences</b>
{% endif %}
</div>
```

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**